transmitter.js

```diff
@@ -2,15 +2,14 @@ let transmissionFailureCount = 0;
 
 function networkTransmitStub(celcius) {

```

Comment:
`transmitInCelcius` is production code, but it also has the stub in the same file. Stub is required only for testing. Move the stub to the test file. Then you'll get a situation where `transmitInCelcius` requires the stub in test file, and test file needs `transmitInCelcius` to call it. To get ideas for solving, look up "dependency injection"